src/index.js

```diff
@@ -254,33 +300,96 @@ function addDatePicker() {
   });
 }
 
-function bookRoom() {
-  let roomNumP = $("#customer-selection-container").children().children().children('p.availability-room-p')[0].innerText;
-  let roomNum = Number(roomNumP.split(' ')[1]);
-  let date = $("#datepicker").val();
-  customer.postBooking(date, roomNum)
-  .then(response => {
-    if (response.ok) {
-      return response.json()
-    } else {
-      return Promise.reject(`error ${response.status} - ${response.statusText}`);
-    }
-  })
-  .then(data => {
-    $("#customer-selection-container").html(`<p class="post-message">Booking Successful!</p>`);
-    setTimeout(function() {$("#customer-selection-container").empty()}, 3000);
-    getData('bookings/bookings', 'bookings')
+function bookRoomManager() {
+  if ($("#customer-selection-container").html().length > 10) {
+    let roomNumP = $("#customer-selection-container").children().children().children('p.availability-room-p')[0].innerText;
+    let roomNum = Number(roomNumP.split(' ')[1]);
+    let date = $("#datepicker").val();
```

Comment:
Chaining on those children()s could be another place to refactor. I know there is the nth-child method for jquery that may allow you to more quickly burrow to the element you want, but it might be better suited for a long li list than nested children.